core-bundle/contao/drivers/DC_Table.php

```diff
@@ -2170,7 +2171,11 @@ public function edit($intId=null, $ajaxId=null)
 				$objTemplate->href = Environment::get('requestUri');
 				$objTemplate->button = $GLOBALS['TL_LANG']['MSC']['continue'];
 
-				throw new ResponseException($objTemplate->getResponse());
+				// We need to set the status code to either 422 or 500 in order for Turbo to render this response.
+				$response = $objTemplate->getResponse();
+				$response->setStatusCode(Response::HTTP_INTERNAL_SERVER_ERROR);
```

Comment:
```suggestion
				$response->setStatusCode(Response::HTTP_UNPROCESSABLE_ENTITY);
```
Could use 422 here since that one describes the problem the best

Reply:
Neither really fits. The entity was in fact processed at this point. Turbo just forces us to use either one.

Reply:
I think `422` is more appropriate in this case.

<img width="691" alt="" src="https://github.com/user-attachments/assets/fd1b4f29-1b9c-47ae-9baa-294343c0fa6b">

Reply:
But did the user mess up? It's not his fault that another user happened to edit the record at the same time 😁

Reply:
I agree with @fritzmg because of this: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422
> Clients that receive a `422` response should expect that repeating the request without modification will fail with the same error.

Since Turbo only allows either `422` and `500` to handle this error, the `500` is more correct considering it's our fault for *allowing* to edit an entry whilst someone else is within edit-mode.

Reply:
> But did the user mess up? It's not their fault that another user happened to edit the record at the same time 😁

True, but it certainly is not a server error. The server and the software both work just fine, don‘t they?

> Clients that receive a `422` response should expect that repeating the request without modification will fail with the same error.

Which is the case, isn‘t it?

Reply:
Yes, resending the same exact request would result in the same response.

Reply:
Wouldn't it be best if the `be_conflict` screen would appear every time with the identical POST data until some extra flag (confirming the action) was set? Then the outcome would be reproducible and also 422 would be correct. 

If that's too complicated, I would go with 422. The status code is not _that_ important in this case. It's only the API for our own front end, there will never be an additional layer.

Reply:
Judging by the code, any 4xx status code should work. So what about [409](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409)?

Reply:
I like 409 👍

Reply:
Changed to 409 👍